manifests/charts/istio-control/istio-discovery/templates/revision-tags.yaml

```diff
@@ -0,0 +1,61 @@
+{{- range $tagName := $.Values.revisionTags }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: istio-revision-tag-{{ $tagName }}
+  labels:
+    istio.io/rev: {{ $.Values.revision | default "default" }}
+    istio.io/tag: {{ $tagName }}
+    install.operator.istio.io/owning-resource: {{ $.Values.ownerName | default "unknown" }}
+    operator.istio.io/component: "Pilot"
+    app: sidecar-injector
+webhooks:
+  - name: sidecar-injector.istio.io
+    clientConfig:
+      {{- if $.Values.istiodRemote.injectionURL }}
+      url: {{ $.Values.istiodRemote.injectionURL }}
+      {{- else }}
+      service:
+        name: istiod{{- if not (eq $.Values.revision "") }}-{{ $.Values.revision }}{{- end }}
+        namespace: {{ $.Release.Namespace }}
+        path: "/inject"
+      {{- end }}
+      caBundle: ""
+    sideEffects: None
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["pods"]
+    failurePolicy: Fail
+    admissionReviewVersions: ["v1beta1", "v1"]
+    namespaceSelector:
+      matchExpressions:
+      - key: istio-injection
+        operator: DoesNotExist
+      - key: istio.io/rev
+        operator: In
+        values:
+        - {{ $.Values.revision }}
+{{- if $.Values.sidecarInjectorWebhook.objectSelector.enabled }}
+    objectSelector:
+{{- if $.Values.sidecarInjectorWebhook.objectSelector.autoInject }}
+      matchExpressions:
+      - key: "sidecar.istio.io/inject"
+        operator: NotIn
+        values:
+        - "false"
+{{- else if $.Values.revision }}
+      matchExpressions:
+      - key: "sidecar.istio.io/inject"
+        operator: DoesNotExist
+      - key: istio.io/rev
+        operator: In
+        values:
+        - {{ $.Values.revision }}
```

Comment:
this should be $tag not .values.revision I think?

Reply:
and asme above

Reply:
Thanks, good catch. Fixed and changed the tests to check that in the future 🙃